README.md

```diff
@@ -38,9 +51,11 @@ Expected a header file
 The repository also contains a comprehensive set of tests, written using the [jtreg](https://openjdk.java.net/jtreg/) test framework, which can be run as follows (again, on Windows, `gradlew.bat` should be used instead):
 
 ```sh
-$ sh ./gradlew -Pjdk18_home=<jdk18_home_dir> -Pllvm_home=<libclang_dir> jtreg
+$ sh ./gradlew -Pjdk18_home=<jdk18_home_dir> -Pllvm_home=<libclang_dir> -Pjtreg_home=<jtreg_home> jtreg
```

Comment:
I noticed `jtreg` task also required this property to be set.

However I do have a question about which version of jtreg should be used, as a convenience I downloaded jtreg6 from Aleksey Shipilëv https://builds.shipilev.net/jtreg/

Reply:
that version of jtreg should be ok

Reply:
Sorry, I think it's better not to have the link (I should have made that clearer).

Reply:
No problem, that was what I assumed there.